### PR TITLE
Adding some view convenience functions

### PR DESCRIPTION
**Pull request by BryanDeemer**
Adding a few convenience functions I've been using for 6 years and I'm not sure I can live without them!

### CODE REVIEW

File: Sources/URBNSwiftyConvenience/UIView+Additions.swift [hunk 1]
```diff
@@ -158,4 +158,35 @@ extension UIView {
         // Alert the layout needs updating
         setNeedsLayout()
     }
+    
+    func sizeToWidthOfContainer(){
+        sizeToWidthOfContainerWithEqualInset(0)
+    }
+    
+    func sizeToWidthOfContainerWithEqualInset(_ inset: CGFloat){
```
Comment (ryang1428) [inline Sources/URBNSwiftyConvenience/UIView+Additions.swift:166]: @BryanDeemer thoughts on this naming?
`func sizeToWidthOfContainer(withEqualInset inset: CGFloat)`

**Review by ryang1428 — CHANGES_REQUESTED** [review]

File: Sources/URBNSwiftyConvenience/UIView+Additions.swift (hunk 1 above)
Comment (BryanDeemer) [reply Sources/URBNSwiftyConvenience/UIView+Additions.swift:166]: That's fine with me. I'll change it now

File: Sources/URBNSwiftyConvenience/UIView+Additions.swift [hunk 2]
```diff
@@ -158,4 +158,35 @@ extension UIView {
         // Alert the layout needs updating
         setNeedsLayout()
     }
+    
+    func sizeToWidthOfContainer(){
+        sizeToWidthOfContainerWithEqualInset(0)
+    }
+    
+    func sizeToWidthOfContainer(WithEqualInset inset: CGFloat){
```
Comment (ryang1428) [inline Sources/URBNSwiftyConvenience/UIView+Additions.swift:166]: small `with`

Comment (BryanDeemer) [reply Sources/URBNSwiftyConvenience/UIView+Additions.swift]: Doh! That was a stupid thing for me to overlook. Thanks for catching it. Fixed.

**Review by ryang1428 — APPROVED** [review]

**Review by tahagabre — APPROVED** [review]

**Pull request merged**